### DISCUSSION TITLE
Spread the word about Deta on indiehacker

### DISCUSSION THREAD

**Discussion by tongsangcong** [deta/cloud-docs#246] [General] [answered]
hello Deta team, I'm using Deta for more than 6+ months now, it's super amazing, I build my product MVP using Deta, every time. Would you guys consider to spread the word about Deta on indiehacker too? indiehacker is like a product community, where indie makers like me learn and share, they should know how amazing the Deta is (I never look up to MongoDB or Heroku anymore ever since I try Deta)

actually I also recommend makers there to try Deta, maybe you guys should active in there too since it's a lot of startup founder and makers who's learn, teach, and share in there

*thank you for creating an amazing product for us

**Comment by abdelhai** [comment, marked as answer]
hey, thank you very much for using deta and sharing your thoughts!

indiehackers is indeed an interesting platform, we even used to announce our products there in the early days.
currently, we try to not be overly pushy with our messages, and let devs discover us and share their honest words.

soon, we will be sharing more stuff on our own channels and hopefully, we will be able to collaborate with other dev-oriented websites.
